### PR TITLE
S1

### PR DESCRIPTION
**Pull request by agondi**
added starter project #1

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,18 +1,38 @@
...
+     <div class="centered"> 
+         <div class="w3-container w3-center w3-animate-opacity">
+          <a id="but1" href="map.html" class="hidden">
+     <button type="button" class="btn btn-dark">Where I've been on the map</button>
+ </a> 
+        <br>
+        <br>
+                 <img id="leftarrow" class="hidden" src="uparrow.png" width="15%" height="15%">
+                 <br>
+                 <br>
```
Comment (sakshia125) [inline portfolio/src/main/webapp/index.html:20]: Correct indentation on the file.

**Comment by sakshia125** [comment]
Could you add a description of what you are doing in this starter projects, and attach some images of how the final pages look like?

**Comment by agondi** [comment]
Hey Sakshi! Might help to check out the video in the readme

Amita Gondi
B.S. Candidate | Computer Science
Minor Candidate | Theatre and Performing Studies (TAPS)
Stanford University | Class of 2022
+1 (650) 387-4666 | agondi@stanford.edu<mailto:mallikaj@stanford.edu>
pronouns: she, her, hers

________________________________

**Comment by agondi** [comment]
but will also add a description.

Amita Gondi
B.S. Candidate | Computer Science
Minor Candidate | Theatre and Performing Studies (TAPS)
Stanford University | Class of 2022
+1 (650) 387-4666 | agondi@stanford.edu<mailto:mallikaj@stanford.edu>
pronouns: she, her, hers

________________________________

**Pull request closed without merging**